spec/support/webmock.rb

```diff
@@ -0,0 +1,3 @@
+require 'webmock/rspec'
+
+WebMock.allow_net_connect!
```

Comment:
Are you sure that we want [this behaviour](https://github.com/bblimke/webmock#real-requests-to-network-can-be-allowed-or-disabled)? I think I'd rather know if any tests are trying to use the network by a failure...

Reply:
Ideally I don't want this behaviour, but at the moment some other tests are making live HTTP requests to the id-mapping-store, and stubbing those correctly seemed like a bit of a rabbit hole!

I've just opened https://github.com/mysociety/verification-pages/issues/112 so that this doesn't get forgotten about.